content/applications/sales/point_of_sale.rst

```diff
@@ -1,5 +1,4 @@
 :show-content:
-:show-toc:
```

Comment:
Do you delete this because it looks ugly?
If so, leave it there.

Don't break the normal navigation of the whole documentation just on this page, for now, although I agree we could reconsider if displaying the toc tree like it is done now is still relevant.

Reply:
I've chosen the "cards" suggestion, it looks fine :)